### PR TITLE
Adds Spring java configuration

### PR DESCRIPTION
**Pull request by rsertelon**
This is just for the sake of clearing all XML config from our projects ;)

### CODE REVIEW

File: src/main/java/com/excilys/ebi/utils/spring/log/SpringLogConfig.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+import com.excilys.ebi.utils.spring.log.slf4j.InjectLoggerAnnotationBeanPostProcessor;
+
+@Configuration
+@ComponentScan(basePackageClasses = InjectLoggerAnnotationBeanPostProcessor.class)
+public class SpringLogConfig {
+
+}
```
Comment (slandelle) [inline src/main/java/com/excilys/ebi/utils/spring/log/SpringLogConfig.java]: missing empty line

Comment (rsertelon) [inline src/main/java/com/excilys/ebi/utils/spring/log/SpringLogConfig.java]: To the end of the file?

Comment (slandelle) [inline src/main/java/com/excilys/ebi/utils/spring/log/SpringLogConfig.java]: Yep

Comment (slandelle) [inline src/main/java/com/excilys/ebi/utils/spring/log/SpringLogConfig.java]: Standard git convention

**Comment by rsertelon** [comment]
Fixed ;)

**Pull request merged**